helpers.h

```diff
@@ -1,16 +1,17 @@
 // To be Included the header file "bmp.h"
+#include "bmp.h"
 
 // Convert Image to grayscale
-// write protopype below this comment
+  void grayscale(int ,int ,RGBTRIPLE);
```

Comment:
Expected a double pointer in third argument i.3 `void grayscale(in, int, RGbTRIPLE **);`